CUE4Parse/UE4/Assets/Exports/StaticMesh/FStaticMeshRenderData.cs

```diff
@@ -24,10 +24,11 @@ public FStaticMeshRenderData(FAssetArchive Ar, bool bCooked)
             if (!bCooked) return;
 
             // TODO Read minMobileLODIdx only when platform is desktop and CVar r.StaticMesh.KeepMobileMinLODSettingOnDesktop is nonzero
-            if (Ar.Versions["StaticMesh.KeepMobileMinLODSettingOnDesktop"])
-            {
-                var minMobileLODIdx = Ar.Read<int>();
-            }
+            // Skip this to fix UE4.27 StaticMesh serialization
+            //if (Ar.Versions["StaticMesh.KeepMobileMinLODSettingOnDesktop"])
+            //{
+            //    var minMobileLODIdx = Ar.Read<int>();
+            //}
```

Comment:
this can be controlled in your fmodel under Settings > Versioning Config > Options
commenting it will just break all other games

Reply:
oh, good to know, didn't realize it can be changed via options